server/vehicle/main.lua

```diff
@@ -204,5 +206,24 @@ function Ox.GeneratePlate()
     end
 end
 
+---Creates a unique vehicle vin number.
+---@return string
+function Ox.GenerateVin()
+    local vin = table.create(17, 0)
+    while true do
+        ::start::
```

Comment:
Why is there a label? Assuming this is just based on the plate generation, not sure why this is added.